hawkrest/__init__.py

```diff
@@ -92,11 +115,14 @@ def authenticate(self, request):
         # Pass our receiver object to the middleware so the request header
         # doesn't need to be parsed again.
         request.META['hawk.receiver'] = receiver
-        return on_success
+        return self.hawk_user_lookup(request, receiver.resource.credentials)
 
     def authenticate_header(self, request):
         return 'Hawk'
 
+    def get_user(self, user_id):
+        return HawkAuthenticatedUser()
```

Comment:
Since this isn't an official DRF hook I think it would help to add a comment saying that this was added for Django compatibility.

Also, I think it would be helpful to add a quick test that simply asserts this method returns a `HawkAuthenticatedUser`. This will aid in future refactoring and protect it from bitrotting accidentally.